### ISSUE TITLE
Allow customizing the 'What we do" section

### ISSUE THREAD

**Issue by Splaktar** [gdg-x/aura#3]
It currently has icons for Angular, Google Cloud, Firebase, Android, Tensorflow, and Assistant.

1. It would be nice to be able to customize this list per chapter.
1. The icons should have tooltips and alt text that is customizable.

**Comment by Vrijraj** [comment]
New Features added
1. Tooltips and alt text 
2. Can customize the icon from [json](https://github.com/gdg-x/aura/blob/master/src/assets/data/whatWeDoComponentData.json) file

and  the component: https://github.com/gdg-x/aura/blob/master/src/components/home/whatWeDo.vue

**Comment by Splaktar** [comment]
Looks good!

I know that this wasn't mentioned before, but it might be useful to optionally allow a url to be provided to link the icon to the docs page of the technology.

**Comment by Vrijraj** [comment]
URL added.
Need your review.
https://aura-client-master.firebaseapp.com/
and component: https://github.com/gdg-x/aura/blob/master/src/components/home/whatWeDo.vue

**Comment by Splaktar** [comment]
Looks good, but I would call the argument here something like `url` or `link` rather than `pic`:
https://github.com/gdg-x/aura/blob/82f8494c19631c79d3d92eae24f421affa36a8c1/src/components/home/whatWeDo.vue#L72-L74

**Comment by Vrijraj** [comment]
I've made the changes. Have a look at that 

https://github.com/gdg-x/aura/blob/master/src/components/home/whatWeDo.vue

**Comment by Splaktar** [comment]
LGTM!
